weatherman.py

```diff
@@ -1,19 +1,61 @@
+import os
+import glob
 import sys
-sys.argv
 
+sys.argv
 for w in sys.argv:
     print(w)
+if len(sys.argv) < 3:
+    sys.exit(1)
 
 my_weatherlist = []
 
-def func_access(file_name):
-    file = open (file_name , "r+")
-    read = file.read().splitlines()
+def read_files(file_name):
+    file = open (file_name , "r")
+    read = file.read().splitlines(",")
```

Comment:
You don't need split lines on `,`. Here `splitlines()` split file into lines and we don't need to split it comma based. We only need it when we are splitting content in a line.